h/views/activity.py

```diff
@@ -88,17 +90,22 @@ def user_link(userid):
         }
 
 
-@view_defaults(route_name='activity.group_search',
-               renderer='h:templates/activity/search.html.jinja2')
+@view_defaults(route_name='group_read',
+               renderer='h:templates/activity/search.html.jinja2',
+               effective_principals=security.Authenticated,
+               has_feature_flag='search_page',
+               has_permission='read')
```

Comment:
Yes, this is nice - although as per the discussion in #backend earlier, eliminating the activity pages feature flag ASAP will simplify the possible combinations somewhat.